WooCommerce/Classes/ViewRelated/Orders/OrdersMasterViewController.swift

```diff
@@ -107,6 +74,36 @@ final class OrdersMasterViewController: UIViewController {
 
         present(navigationController, animated: true, completion: nil)
     }
+
+    // MARK: - ButtonBarPagerTabStripViewController Conformance
+
+    /// Return the ViewControllers for "Processing" and "All Orders".
+    ///
+    override func viewControllers(for pagerTabStripController: PagerTabStripViewController) -> [UIViewController] {
+        // TODO This is fake. It's probably better to just pass the `slug` to `OrdersViewController`.
+        let processingOrderStatus = OrderStatus(
+            name: OrderStatusEnum.processing.rawValue,
+            siteID: Int64.max,
+            slug: OrderStatusEnum.processing.rawValue,
+            total: 0
+        )
```

Comment:
do we plan on addressing this TODO in a followup task? feel free to refactor `OrderStatus`'s init as a separate task

Reply:
Yes. It's on my list. I also added it to https://github.com/woocommerce/woocommerce-ios/issues/956.